Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -89,6 +89,18 @@ int RGroupDecomposition::add(const ROMol &inmol) {
   const bool addCoords = true;
   MolOps::addHs(mol, explicitOnly, addCoords);
 
+  // mark any wildcards in input molecule:
+  for (auto &atom : mol.atoms()) {
+    if (atom->getAtomicNum() == 0) {
+      atom->setProp("INPUT_DUMMY", true);
```

Comment:
Definitely not required, but it would be better if you had a constexpr for `"INPUT_DUMMY"`

Reply:
Made that change